### ISSUE TITLE
Start using EditorConfig in source

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2427]
Originally reported on Google Code with ID 2162
```
"EditorConfig helps developers define and maintain consistent coding styles between
different editors and IDEs. The EditorConfig project consists of a file format for
defining coding styles and a collection of text editor plugins that enable editors
to read the file format and adhere to defined styles. EditorConfig files are easily
readable and they work nicely with version control systems."

http://editorconfig.org/

Pull requests via @treyhunner:
https://github.com/arshaw/fullcalendar/pull/141
https://github.com/arshaw/fullcalendar/pull/142

```

Reported by `adamrshaw` on 2014-06-05 20:42:21

Imported with **1** stars.

**Comment by arshaw** [comment]
i understand the need for this. would entail merging #141

**Comment by arshaw** [comment]
merged in 2.8.0
